### DISCUSSION TITLE
Elm_Creek_Edit_4x_By_Stevie_

### DISCUSSION THREAD

**Discussion by aurelien2023** [KillBait/FS19_AutoDrive_MapImages#11] [General]
Here the map image for Elm_Creek_Edit_4x_By_Stevie_
[](url
![Elm_Creek_Edit_4x_By_Stevie_](https://user-images.githubusercontent.com/60996361/179529354-28aeb3b8-3fb5-42e2-8e41-ceeaa423e44a.png)
)

**Comment by KillBait** [comment]
Added to the List, editor should download it automatically
